scientific/src/commonMain/kotlin/converter/area/convertFromKinematicViscosityAndTime.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ Copyright 2021 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.scientific.converter.area
+
+import com.splendo.kaluga.base.utils.Decimal
+import com.splendo.kaluga.scientific.DefaultScientificValue
+import com.splendo.kaluga.scientific.PhysicalQuantity
+import com.splendo.kaluga.scientific.ScientificValue
+import com.splendo.kaluga.scientific.byMultiplying
+import com.splendo.kaluga.scientific.unit.Area
+import com.splendo.kaluga.scientific.unit.KinematicViscosity
+import com.splendo.kaluga.scientific.unit.Time
+import kotlin.jvm.JvmName
+
+@JvmName("areaFromKinematicViscosityAndTimeDefault")
+fun <
```

Comment:
I'm not familiar with Kaluga coding style, but this looks quite strange to my 👀 
Anyways, please just make sure this follows style/formatting of other places whatever that is

Reply:
I double checked with other converters and the coding style for these functions match. also double checked with ktlint